### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by olawale-o**
# This pull request contains:
- /src directory(index.html, index.js, /stylesheets(style.css))
- webpack.config.js file
- .github/workflow directory(linters.yml)
- package.json file
- package-lock.json
- .hintrc file
- .stylelintrc.json file
- .eslintrc.json file
- README.md

## The following was implemented:
- Create an npm project with webpack.
- Create plain HTML markup of the wireframe with minimum styling without hitting the API

### CODE REVIEW

**Review by vichuge — CHANGES_REQUESTED** [review]
# Project invalidated 🚫 

Hi there, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

cat_massage

## Why does this happen ❓ ❓ ❓ 

You have submitted a project, that is not using `main` as a default branch. I kindly invite you to use main as the default branch.
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by olawale-o** [comment]
Hi @vichuge 

I really appreciate your time for reviewing my task you were clear and concise about the issue. 
- The task requires us to use Gitflow
- From the Gitflow we are required to create a dev or (develop or development branch)
- Then all branches created should track the dev or (develop or development branch)
- Kindly find below a screenshot of the requirement which specifies Gitflow
Screen Shot 2021-07-26 at 14 42 52

**Comment by vichuge** [comment]
> Hi @vichuge
> 
> I really appreciate your time for reviewing my task you were clear and concise about the issue.
> 
> * The task requires us to use Gitflow
> * From the Gitflow we are required to create a dev or (develop or development branch)
> * Then all branches created should track the dev or (develop or development branch)
> * Kindly find below a screenshot of the requirement which specifies Gitflow
>   Screen Shot 2021-07-26 at 14 42 52

I just read the requirements and you're right. Sorry for the misunderstood.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "build": "webpack",
+    "start": "webpack serve --open",
+    "watch": "webpack --watch",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "eslint": "eslint .",
+    "stylelint": "stylelint \"**/*.{css,scss}\""
+  },
```
Comment (vichuge) [inline package.json]: - [ ] Please, add the next option here `"dev": "webpack --mode development",`.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                            <div>
+                                <button type="submit" class="btn" id="add-score">Submit</button>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+</body>
```
Comment (vichuge) [inline src/index.html]: - [ ] index.html main contain only root element.
- Hint: At this point, the elements for the project should be made with js and the `index.html` should have only the `<div>` where you build your page from js. For example, you could add a div like `<div id="root"></div>` and then, call this `<div>` on index.js and add all elements with js code.
- When you build all this with js on index.js, try to use ES6 modules to write modular JavaScript with proper ES6 syntax.

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- [OPTIONAL]

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- [OPTIONAL]

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @olawale-o, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (Dipeshtwis) [reply package.json]: Applied changes successfully :+1:

File: src/index.html (hunk 2 above)
Comment (Dipeshtwis) [reply src/index.html]: Applied changes successfully :+1:

**Pull request merged**